corehq/apps/app_manager/templates/app_manager/v2/partials/module_view_heading.html

```diff
@@ -8,10 +8,10 @@
   {% endif %}
 </div>
 <div class="appmanager-edit-title">
-  {% if module.case_type == '' %}
-      {% inline_edit_trans_v2 module.name langs edit_name_url saveValueName='name' containerClass='h3' iconClass='fa fa-folder-open' postSave="function(data) { return hqImport('app_manager/js/app_manager.js').updateDOM(data.update); }" %}
-  {% else %}
+  {% if module.is_surveys %}
```

Comment:
I think these are reversed, right?  `module.is_surveys` == `module.case_type == ''`, but the else block has been swapped.